### ISSUE TITLE
No ouput transforming ADE

### ISSUE THREAD

**Issue by PRemmen** [3dcitydb/plugin-ade-manager#6]
I'm just starting to use the ADE-manager of 3DcityDB. I can register the https://github.com/3dcitydb/extension-test-ade using the mapping-xml from: extension-test-ade/test-ade-citydb/resources/

However, I am not able to generate my own schema-mapping.xml and thus, I'm not able to register other ADEs (in my case EnergyADE and UtilityNetworksADE)

![grafik](https://user-images.githubusercontent.com/8124013/52340589-6d1d7080-2a10-11e9-8a1a-d0bbb8db82b3.png)

Here is the log:

```
[12:58:47 INFO] Transforming ADE's XML schema to relational database schema...
[12:58:48 INFO] Matched transformation rule applied. (Step: 1)
[12:58:48 INFO] Matched transformation rule applied. (Step: 2)
[12:58:48 INFO] Matched transformation rule applied. (Step: 3)
[12:58:48 INFO] Matched transformation rule applied. (Step: 4)
[12:58:48 INFO] Matched transformation rule applied. (Step: 5)
[12:58:48 INFO] Matched transformation rule applied. (Step: 6)
[12:58:48 INFO] Matched transformation rule applied. (Step: 7)
[12:58:48 INFO] Matched transformation rule applied. (Step: 8)
[12:58:48 INFO] Matched transformation rule applied. (Step: 9)
[12:58:48 INFO] Matched transformation rule applied. (Step: 10)
[12:58:48 INFO] Matched transformation rule applied. (Step: 11)
[12:58:48 INFO] Matched transformation rule applied. (Step: 12)
[12:58:48 INFO] Matched transformation rule applied. (Step: 13)
[12:58:48 INFO] Matched transformation rule applied. (Step: 14)
[12:58:48 INFO] Matched transformation rule applied. (Step: 15)
[12:58:48 INFO] Matched transformation rule applied. (Step: 16)
[12:58:48 INFO] Matched transformation rule applied. (Step: 17)
[12:58:48 INFO] Matched transformation rule applied. (Step: 18)
[12:58:48 INFO] Matched transformation rule applied. (Step: 19)
[12:58:48 INFO] Matched transformation rule applied. (Step: 20)
[12:58:48 INFO] Matched transformation rule applied. (Step: 21)
[12:58:48 INFO] Matched transformation rule applied. (Step: 22)
[12:58:48 INFO] Matched transformation rule applied. (Step: 23)
[12:58:48 INFO] Matched transformation rule applied. (Step: 24)
[12:58:48 INFO] Matched transformation rule applied. (Step: 25)
[12:58:48 INFO] Matched transformation rule applied. (Step: 26)
[12:58:48 INFO] Matched transformation rule applied. (Step: 27)
[12:58:48 INFO] Matched transformation rule applied. (Step: 28)
[12:58:48 INFO] Matched transformation rule applied. (Step: 29)
[12:58:48 INFO] Matched transformation rule applied. (Step: 30)
[12:58:48 INFO] Matched transformation rule applied. (Step: 31)
[12:58:48 INFO] Matched transformation rule applied. (Step: 32)
[12:58:48 INFO] Matched transformation rule applied. (Step: 33)
[12:58:48 INFO] Matched transformation rule applied. (Step: 34)
[12:58:48 INFO] Matched transformation rule applied. (Step: 35)
[12:58:48 INFO] Matched transformation rule applied. (Step: 36)
[12:58:48 INFO] Matched transformation rule applied. (Step: 37)
[12:58:48 INFO] Matched transformation rule applied. (Step: 38)
[12:58:48 INFO] Matched transformation rule applied. (Step: 39)
[12:58:48 INFO] Matched transformation rule applied. (Step: 40)
[12:58:48 INFO] Matched transformation rule applied. (Step: 41)
[12:58:48 INFO] Matched transformation rule applied. (Step: 42)
[12:58:48 INFO] Matched transformation rule applied. (Step: 43)
[12:58:48 INFO] Matched transformation rule applied. (Step: 44)
[12:58:48 INFO] Matched transformation rule applied. (Step: 45)
[12:58:48 INFO] Matched transformation rule applied. (Step: 46)
[12:58:48 INFO] Matched transformation rule applied. (Step: 47)
[12:58:48 INFO] Matched transformation rule applied. (Step: 48)
[12:58:48 INFO] Matched transformation rule applied. (Step: 49)
[12:58:48 INFO] Matched transformation rule applied. (Step: 50)
[12:58:48 INFO] Matched transformation rule applied. (Step: 51)
[12:58:48 INFO] Matched transformation rule applied. (Step: 52)
[12:58:48 INFO] Matched transformation rule applied. (Step: 53)
[12:58:48 INFO] Matched transformation rule applied. (Step: 54)
[12:58:48 INFO] Matched transformation rule applied. (Step: 55)
[12:58:48 INFO] Matched transformation rule applied. (Step: 56)
[12:58:48 INFO] Matched transformation rule applied. (Step: 57)
[12:58:48 INFO] Matched transformation rule applied. (Step: 58)
[12:58:48 INFO] Matched transformation rule applied. (Step: 59)
[12:58:48 INFO] Matched transformation rule applied. (Step: 60)
[12:58:48 INFO] Matched transformation rule applied. (Step: 61)
[12:58:48 INFO] Matched transformation rule applied. (Step: 62)
[12:58:48 INFO] Matched transformation rule applied. (Step: 63)
[12:58:48 INFO] Matched transformation rule applied. (Step: 64)
[12:58:48 INFO] Matched transformation rule applied. (Step: 65)
[12:58:48 INFO] Matched transformation rule applied. (Step: 66)
[12:58:48 INFO] Matched transformation rule applied. (Step: 67)
[12:58:48 INFO] Matched transformation rule applied. (Step: 68)
[12:58:49 INFO] Matched transformation rule applied. (Step: 69)
[12:58:49 INFO] Matched transformation rule applied. (Step: 70)
[12:58:49 INFO] Matched transformation rule applied. (Step: 71)
[12:58:49 INFO] Matched transformation rule applied. (Step: 72)
[12:58:49 INFO] Matched transformation rule applied. (Step: 73)
[12:58:49 INFO] Matched transformation rule applied. (Step: 74)
[12:58:49 INFO] Matched transformation rule applied. (Step: 75)
[12:58:49 INFO] Matched transformation rule applied. (Step: 76)
[12:58:49 INFO] Matched transformation rule applied. (Step: 77)
[12:58:49 INFO] Matched transformation rule applied. (Step: 78)
[12:58:49 INFO] Matched transformation rule applied. (Step: 79)
[12:58:49 INFO] Matched transformation rule applied. (Step: 80)
[12:58:49 INFO] Matched transformation rule applied. (Step: 81)
[12:58:49 INFO] Matched transformation rule applied. (Step: 82)
[12:58:49 INFO] Matched transformation rule applied. (Step: 83)
[12:58:49 INFO] Matched transformation rule applied. (Step: 84)
[12:58:49 INFO] Matched transformation rule applied. (Step: 85)
[12:58:49 INFO] Matched transformation rule applied. (Step: 86)
[12:58:49 INFO] Matched transformation rule applied. (Step: 87)
[12:58:49 INFO] Matched transformation rule applied. (Step: 88)
[12:58:49 INFO] Matched transformation rule applied. (Step: 89)
[12:58:49 INFO] Matched transformation rule applied. (Step: 90)
[12:58:49 INFO] Matched transformation rule applied. (Step: 91)
[12:58:49 INFO] Matched transformation rule applied. (Step: 92)
[12:58:49 INFO] Matched transformation rule applied. (Step: 93)
[12:58:49 INFO] Matched transformation rule applied. (Step: 94)
[12:58:49 INFO] Matched transformation rule applied. (Step: 95)
[12:58:49 INFO] Matched transformation rule applied. (Step: 96)
[12:58:49 INFO] Matched transformation rule applied. (Step: 97)
[12:58:49 INFO] Matched transformation rule applied. (Step: 98)
[12:58:49 INFO] Matched transformation rule applied. (Step: 99)
[12:58:49 INFO] Matched transformation rule applied. (Step: 100)
[12:58:49 INFO] Matched transformation rule applied. (Step: 101)
[12:58:49 INFO] Matched transformation rule applied. (Step: 102)
[12:58:49 INFO] Matched transformation rule applied. (Step: 103)
Used time for graph transformation:  1504ms
Exception in thread "Thread-36" java.lang.NoSuchMethodError: com.sun.org.apache.xml.internal.serialize.OutputFormat.<init>(Lorg/w3c/dom/Document;Ljava/lang/String;Z)V
	at agg.util.XMLHelper.save_to_xml(XMLHelper.java:137)
	at org.citydb.plugins.ade_manager.transformation.graph.GraphTransformationManager.marshallingEdGraphGrammer(GraphTransformationManager.java:127)
	at org.citydb.plugins.ade_manager.transformation.graph.GraphTransformationManager.executeGraphTransformation(GraphTransformationManager.java:90)
	at org.citydb.plugins.ade_manager.transformation.TransformationController.doProcess(TransformationController.java:70)
	at org.citydb.plugins.ade_manager.gui.modules.ADETransformationPanel.transformADESchema(ADETransformationPanel.java:458)
	at org.citydb.plugins.ade_manager.gui.modules.ADETransformationPanel.access$300(ADETransformationPanel.java:72)
	at org.citydb.plugins.ade_manager.gui.modules.ADETransformationPanel$5$1.run(ADETransformationPanel.java:229)
```

I'm using Ubuntu 18.04 and following version of java:

```
$ java -version
   openjdk version "10.0.2" 2018-07-17
   OpenJDK Runtime Environment (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4)
   OpenJDK 64-Bit Server VM (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.4, mixed mode)
```

Am I doing something wrong? __Any__ help highly appreciated @yaozhihang

**Comment by yaozhihang** [comment]
Hi @PRemmen, thanks for the bug report, i can also reproduce this issue on my windows machine with **OpenJDK** 11. 
Using Oracle JDK/JRE may resolve this issue. But i'll try to fix this in order to also work with OpenJDK.

**Comment by PRemmen** [comment]
perfect! on windows with:

```
java -jar -version
java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)
```
it works! :) thanks a lot for your comment!

**Comment by yaozhihang** [comment]
fixed in 2b3753110eca79c9f4357479fe38fc4768c7e47e
You can build the ADE Manager plugin by following the instructions here:
https://github.com/3dcitydb/plugin-ade-manager#installation

**Comment by yaozhihang** [comment]
The fix is now included in a new version of the ADE Manager. It is available here
https://github.com/3dcitydb/plugin-ade-manager/releases
Please note that you need to run this version of ADE Manager with the Importer/Exporter v4.2.0
